doc/install/index.rst

```diff
@@ -1,72 +1,74 @@
+requirements, like ease of maintenance.
+
+
+ceph-ansible
 ============
 
-There are several methods for getting Ceph software. The easiest and most common
-method is to `get packages`_ by adding repositories for use with package
-management tools such as the Advanced Package Tool (APT) or Yellowdog Updater,
-Modified (YUM). You may also retrieve pre-compiled packages from the Ceph
-repository. Finally, you can retrieve tarballs or clone the Ceph source code
-repository and build Ceph yourself.
+Installs Ceph using Ansible.
 
+* `docs.ceph.com/ceph-ansible <https://docs.ceph.com/ceph-ansible/>`_ 
 
-.. toctree::
-   :maxdepth: 1
+ceph-deploy
+===========
 
-	Get Packages <get-packages>
-	Get Tarballs <get-tarballs>
-	Clone Source <clone-source>
-	Build Ceph <build-ceph>
-    	Ceph Mirrors <mirrors>
-	Ceph Containers <containers>
+Install ceph using ceph-deploy
 
+* :ref:`ceph-deploy-index`
 
-Install Software
-================
 
-Once you have the Ceph software (or added repositories), installing the software
-is easy. To install packages on each :term:`Ceph Node` in your cluster. You may
-use  ``ceph-deploy`` to install Ceph for your storage cluster, or use package
-management tools. You should install Yum Priorities for RHEL/CentOS and other
-distributions that use Yum if you intend to install the Ceph Object Gateway or
-QEMU.
+cephadm
+=======
 
-.. toctree::
-   :maxdepth: 1
+Installs Ceph using containers and systemd.
 
-	Install ceph-deploy <install-ceph-deploy>
-    	Install Ceph Storage Cluster <install-storage-cluster>
-	Install Ceph Object Gateway <install-ceph-gateway>
-	Install Virtualization for Block <install-vm-cloud>
+* :ref:`cephadm-bootstrap`
 
+DeepSea
+=======
 
-Deploy a Cluster Manually
-=========================
+Install Ceph using Salt
 
-Once you have Ceph installed on your nodes, you can deploy a cluster manually.
-The manual procedure is primarily for exemplary purposes for those developing
-deployment scripts with Chef, Juju, Puppet, etc.
+* `github.com/SUSE/DeepSea <https://github.com/SUSE/DeepSea>`_
 
-.. toctree::
+Juju
+====
 
-	Manual Deployment <manual-deployment>
-	Manual Deployment on FreeBSD <manual-freebsd-deployment>
+Installs Ceph using Juju.
 
-Upgrade Software
-================
+* `jaas.ai/ceph-mon <https://jaas.ai/ceph-mon>`_
 
-As new versions of Ceph become available, you may upgrade your cluster to take
-advantage of new functionality. Read the upgrade documentation before you
-upgrade your cluster. Sometimes upgrading Ceph requires you to follow an upgrade
-sequence.
 
-.. toctree::
-   :maxdepth: 2
+Manual
+======
+
+Manually install Ceph using packages.
+
+* :ref:`install-manual`
 
-   Upgrading Ceph <upgrading-ceph>
+Puppet
+======
 
-.. _get packages: ../install/get-packages
+Installs Ceph using Puppet
+
+* `github.com/openstack/puppet-ceph <https://github.com/openstack/puppet-ceph>`_
+
+
+Rook
+====
+
+Installs Ceph in Kubernetes.
+
+* `rook.io <https://rook.io/>`_ 
+
```

Comment:
Caveats:
- Rook only supports Nautilus and newer releases.
- Rook is the preferred deployment method for Ceph with Kubernetes.
- Rook fully supports the new orchestrator API, and new management features in the CLI and dashboard are fully supported.

Reply:
s/Rook only supports/Rook supports only
s/the new orchestrator API, and new management features in the CLI and dashboard are fully supported/the new orchestrator API, the new management features in the CLI and dashboard/